Changed docs from cOS to Elemental (#32)
c3os-io/c3os: docs/content/after_install/upgrades.md

## Kubernetes

> Upgrades can be triggered from Kubernetes with `system-upgrade-controller` installed in your cluster. [See the Element documentation](https://rancher.github.io/elemental-toolkit/docs/getting-started/upgrading/#integration-with-system-upgrade-controller)

Reviewer: ```suggestion
Upgrades can be triggered from Kubernetes with `system-upgrade-controller` installed in your cluster. [See the Elemental-toolkit documentation](https://rancher.github.io/elemental-toolkit/docs/getting-started/upgrading/#integration-with-system-upgrade-controller)
```

Use `--force` to force upgrading to avoid checking versions. All the available versions can be list with: `c3os upgrade list-releases`.

> It is possible altough to use the same commandset from `Element`. So for example, the following works too:

Reviewer: ```suggestion
It is possible altough to use the same commandset from `Elemental-toolkit`. So for example, the following works too:
```

```bash
elemental upgrade --no-verify --docker-image quay.io/c3os/c3os:opensuse-v1.21.4-22
```

c3os images are released on [quay.io](https://quay.io/repository/c3os/c3os).

> [See also the general Element documentation](https://rancher.github.io/elemental-toolkit/docs/getting-started/upgrading/#upgrade-to-a-specific-container-image) which applies for `c3os` as well.

Reviewer: ```suggestion
[See also the general Elemental-toolkit documentation](https://rancher.github.io/elemental-toolkit/docs/getting-started/upgrading/#upgrade-to-a-specific-container-image) which applies for `c3os` as well.
```
